docs/aerosim_app.md

```diff
@@ -1,47 +1,163 @@
 
+To visualize and control a simulation through the app, just launch AeroSim enabling the Pixel Streaming, to allow to stream viewport images from the renderer:
+
+- Linux:
 ```sh
-bun install
+./launch_aerosim.sh --unreal-editor --pixel-streaming
 ```
 
-Once the installation has completed, launch the app with the following command from the same :
-
-```
-bun run dev
+- Windows:
+```sh
+launch_aerosim.bat --unreal-editor --pixel-streaming
 ```
 
-You will now see the AeroSim app interface:
+In this example we will use Unreal Editor, but the steps with Omniverse are the same replacing the renderer with `--omniverse`.
 
-![aerosim_app_interface](img/aerosim_app_interface.webp)
+You will now see the AeroSim app interface:
 
-Now launch the AeroSim simulator with the following command:
+![aerosim_app_interface](img/aerosim_app_interface.png)
 
-```sh
-# Windows launch_aerosim.bat --unreal-editor --pixel-streaming
-./launch_aerosim.sh --unreal-editor --pixel-streaming
-```
-
-In the Unreal Engine interface, add a *Google Photorealistic 3D Tiles* asset and choose *Stream level Editor* from the *Pixel Steaming* dropdown menu.
+In the Unreal Engine interface, add a *Google Photorealistic 3D Tiles* asset to enable the Cesium tiles. To enable viewport streaming to the app, choose *Stream level Editor* from the *Pixel Steaming* dropdown menu at the top bar of the editor.
 
 ![stream_level_editor](img/pixel_streaming_option.webp)
 
-Start the simulator in the Unreal Editor using the green play control. Open a separate terminal in the *aerosim* repository, source the AeroSim virtual environment and then enter the `examples/` directory and run the `autopilot_daa_scenario.py` script:
+Start the simulator in the Unreal Editor using the green play control. Open a separate terminal in the *aerosim* repository, source the AeroSim virtual environment and then enter the `examples/` directory and run a script. In this example we will run the `autopilot_daa_scenario.py`:
 
+- Linux:
 ```sh
 source .venv/bin/activate
-cd tutorials/
-# Windows .\.venv\Scripts\activate
-# Windows cd .\tutorials\
+cd examples/
+python autopilot_daa_scenario.py
+```
 
+```sh
+.\.venv\Scripts\activate
+cd examples\
 python autopilot_daa_scenario.py
 ```
 
-Press the *play* symbol on the left hand side of the app to see the ongoing simulation in the app:
+Once the python script is running, press the *reload* symbol on the top left hand side of the app to reload its state and see the ongoing simulation in the app:
+
+![stream_level_editor](img/aerosim_app_daa_scenario.png)
```

Comment:
Let's replace this local image with `https://github.com/aerosim-open/aerosim-app/raw/main/images/app_example.jpg`
![app-repo-example-img](https://github.com/aerosim-open/aerosim-app/raw/main/images/app_example.jpg)
That way, we don't have to add the PNG file again to this repo to save git object space from these binary files and also it'll be easier to update the image in one-place as the App evolves instead of trying to keep these two copies in sync.

Before finalizing this change, please double check to ensure that the `mkdocs` has no issues rendering the remote image URI.